generate-cpio-rootfs.sh

```diff
@@ -219,9 +219,12 @@ if [ ! -e ${BUILDDIR}/.config ]; then
   sed -i -e "s/CONFIG_FEATURE_EJECT_SCSI=y/\# CONFIG_FEATURE_EJECT_SCSI is not set/g" ${BUILDDIR}/.config
   #make O=${BUILDDIR} menuconfig
 fi
-make -j${_NPROCESSORS_ONLN} O=${BUILDDIR}
-make O=${BUILDDIR} install
+make -j${_NPROCESSORS_ONLN} O=${BUILDDIR} && \
+make O=${BUILDDIR} install && \
 cd ${CURDIR}
```

Comment:
Adding a `|| exit 1` would be more compact than the `if` below.
Another option is to add a `set -e` at the beginning of the script to make all unchecked failures abort the script.